go-controller/pkg/ovn/default_network_controller.go

```diff
@@ -770,12 +773,22 @@ func (h *defaultNetworkControllerEventHandler) AddResource(obj interface{}, from
 			if err = h.oc.addUpdateLocalNodeEvent(node, nodeParams); err != nil {
 				klog.Infof("Node add failed for %s, will try again later: %v",
 					node.Name, err)
-				return err
+				errs = append(errs, err)
+			}
+			if err = h.oc.addIPToHostNetworkNamespaceAddrsSet(node); err != nil {
+				h.oc.syncHostNetAddrSetFailed.Store(node.Name, true)
+				errs = append(errs, err)
 			}
+			return kerrors.NewAggregate(errs)
```

Comment:
only if len(errs>0) right? right now will you always be returning an empty array even if there are no errors?

Reply:
NewAggregate will return nil if there are no errors.